src/components/Layout/PageHeader.vue

```diff
@@ -127,6 +127,9 @@ export default {
                 this.$router.push({ name: 'Login' });
                 this.clearCurrent();
             });
+
+            // Clear code id from local storage
+            window.localStorage.clear()
```

Comment:
Since our intention is th remove a specific key from local storage, I'd prefer that we remove only that instead of nuking the whole thing. Otherwise, in the future, somebody may try to store something else, and it'll keep mysteriously disappearing.